src/App.tsx

```diff
@@ -1,19 +1,130 @@
 import React from 'react';
-import './App.css';
-
+import './Todo.scss';
+import { v4 as uuidv4 } from 'uuid';
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './TodoList/TodoList';
+import { UserInfo } from './UserInfo/UserInfo';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+type State = {
+  userName: string,
+  todosCLone: Todo[],
+  newTodo: string,
+  inputError: string,
+  selectError: string,
 };
 
+class App extends React.PureComponent<{}, State> {
+  state:State = {
+    userName: 'chose user',
```

Comment:
```suggestion
    userName: 'Choose user',
```

Install
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker